app/konnect/dev-portal/customization/sso.md

```diff
@@ -0,0 +1,34 @@
+---
+title: SSO / Identity
+no_version: true
+---
+
+Single-Sign-On (a.k.a. SSO) can be configured in the Dev Portal by those with Admin roles via the Identity settings in the {{site.konnect_short_name}} Admin UI. To access the SSO/Identity settings, click **Dev Portal**, then **Settings**, then **Identity**. OpenID Connect (OIDC) allows for Single-Sign-On (SSO) through the Identity Provider (IdP) of their choice.
```

Comment:
Maybe we should add info that it is for enterprise-type only?

Reply:
done!